server/views/pages/activities/manage-activities/view-activity.njk

```diff
@@ -33,7 +33,7 @@
                                 {
                                     href: "/activities/edit/" + activity.id + "/category?preserveHistory=true",
                                     text: "Change",
-                                    visuallyHiddenText: "change category",
```

Comment:
this is a bit irrelevant to the ticket - but we shouldn't be repeating the word 'change' here as it'll read out as "Change change category" to screenreaders